admin/release.py

```diff
@@ -1,4 +1,4 @@
-# -*- test-case-name: admin.test.test_release -*-
+# -*- test-case-name: admin.test.test_release,admin.functional.test_release -*-  # noqa
```

Comment:
:question: I don't have emacs or emacs plugins that parse this, so I'll have to trust your that this is the right form. (Just fyi, I'm sure it's right, but I can't verify)